app/jobs/job_manager.rb

```diff
@@ -0,0 +1,81 @@
+# frozen_string_literal: true
+
+#  Copyright (c) 2023, Schweizer Alpen-Club. This file is part of
+#  hitobito_sac_cas and licensed under the Affero General Public License version 3
+#  or later. See the COPYING file at the top-level directory or at
+#  https://github.com/hitobito/hitobito_sac_cas
+
+class JobManager
+  class_attribute :wagon_jobs, default: []
+
+  def schedule
+    jobs.each do |job_class|
+      job_class.new.schedule
+    end
+  end
+
+  def clear
+    Delayed::Job.delete_all
+  end
+
+  def check
+    scheduled, missing = jobs.partition { _1.new.scheduled? }
```

Comment:
I'll be happy once we have Ruby 3.4 and can write `it.new.scheduled?` here instead.

For now, it is inconsistent with line 12 above. How about:

```suggestion
    scheduled, missing = jobs.partition do |job_class|
      job_class.new.scheduled?
    end
```